src/sqlfluff/core/linter/linter.py

```diff
@@ -491,6 +491,7 @@ def lint_fix_parsed(
 
         save_tree = tree
         for loop in range(loop_limit):
+            linter_logger.info(f"Linter loop {loop+1}/{loop_limit}")
```

Comment:
This was really handy for debugging this issue (which at times caused the linter to hit the loop limit). Seems like a good thing to keep around.